pkg/util/log/eventpb/telemetry.proto

```diff
@@ -769,4 +760,17 @@ message HotRangesStats {
 
   // CPU time per second is the recent cpu usage in nanoseconds of this range.
   double cpu_time_per_second = 13 [(gogoproto.customname) = "CPUTimePerSecond", (gogoproto.jsontag) = ",omitempty"];
+
+  // Databases for the range.
+  repeated string databases = 16;
+  // Tables for the range
+  repeated string tables = 17;
+  // Indexes for the range
+  repeated string indexes = 18;
```

Comment:
So these fields are supposed to replace the ones that were deprecated? Can we get some tests for the new fields?